### PR TITLE
Update readme.txt for 81334

### PR DESCRIPTION
**Pull request by jentheo**

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -122,6 +122,9 @@ Currently, the following add-ons are available for Event Tickets:
 
 == Changelog ==
 
+= [] TBD =
```
Comment (mitogh) [inline readme.txt]: Would you mind add the version here it should be `4.7.5`.

Thanks @jentheo

Comment (jentheo) [reply readme.txt]: Done!

**Comment by mitogh** [comment]
Thank you!

**Comment by juanfra** [comment]
@jentheo would you please merge this PR? Thanks 😄

**Pull request merged**